packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -272,6 +272,14 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+
+  /**
+   * Custom loaded schema for the plugin
+   */
+  schema: {
+    type: Object as PropType<Record<string, any>>,
+    default: null,
+  },
```

Comment:
Should we document this new prop and say this is only for the sandbox?

Reply:
Documented.